polaris-core/src/main/java/org/apache/polaris/core/persistence/pagination/PolarisPage.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.core.persistence.pagination;
+
+import java.util.List;
+
+/**
+ * A wrapper for a {@link List} of data and a {@link PageToken} that can be used to continue the
+ * listing operation that generated that data.
+ */
+public class PolarisPage<T> {
```

Comment:
`Polaris` is superfluous - the type's already in an `org.apache.polaris` package, right?

Reply:
`Polaris` is already the prefix of many class names, but I'm very much in favor of cutting back on that. In this particular case, I worried that `Page` is such a nondescriptive (and widely used) type name that it made sense to make the name explicit that this is a type within Polaris meant to be used only within Polaris.